### ISSUE TITLE
ERROR - Expecting value: line 1 column 1 (char 0)

### ISSUE THREAD

**Issue by broklin48** [roju/tiktok-live-recorder#4]
Today I got an error like this, even though the user is live.
![image](https://github.com/roju/tiktok-live-recorder/assets/122853490/ba3084f9-540e-4b63-98e2-2ecf0200f1eb)

any help?

**Comment by broklin48** [comment]
I think my IP has been blocked or something, I tried using another internet provider, worked fine.
![image](https://github.com/roju/tiktok-live-recorder/assets/122853490/a2b59a4e-4083-4268-9de9-9eecbbd5b6e0)

**Comment by lotaf99** [comment]
yeah i get that issuee too, i think you can raise wait to 5 minute or 300 sec to avoid ip blacklist, you can edit wait time in enum.py

**Comment by lotaf99** [comment]
> > yeah i get that issuee too, i think you can raise wait to 5 minute or 300 sec to avoid ip blacklist, you can edit wait time in enum.py
> 
> did this work for you? I tried it and saw no diference

yeah this is work for me, i didn't get ip blacklist anymore, for 48h running, i didn't get blacklist

**Comment by roju** [comment]
Should the default long wait time be increased to 5 minutes instead of 2 minutes? I have not encountered this error before. Maybe it only affects users from certain countries/locations?
